### PR TITLE
[release-v1.12] Adjust project.yaml to v1.12

### PR DESCRIPTION
**Pull request by creydr**
Adjust project.yaml after branch renaming.

### CODE REVIEW

File: openshift/images.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-knative.dev/eventing-istio/cmd/controller: registry.ci.openshift.org/openshift/knative-eventing-istio-controller:knative-v1.13
+knative.dev/eventing-istio/cmd/controller: registry.ci.openshift.org/openshift/knative-eventing-istio-controller:knative-v1.12
 knative.dev/eventing-kafka-broker/control-plane/cmd/kafka-controller: registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-kafka-controller:knative-v1.13
```
Comment (pierDipi) [inline openshift/images.yaml:2]: Only one?

Comment (creydr) [reply openshift/images.yaml:2]: :thinking: hmmm. I just ran make generate-release

Comment (creydr) [reply openshift/images.yaml:2]: let me recheck

Comment (creydr) [reply openshift/images.yaml]: updated

**Comment by creydr** [comment]
/unhold
@pierDipi images.yaml got updated correctly. Can you recheck?

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**